samples/deploy/crd/v1beta1/ApisixTls.yaml

```diff
@@ -0,0 +1,34 @@
+#
```

Comment:
Have you tried to install this by kustomize?

Reply:
Will test.